x-pack/plugins/observability_ai_assistant/public/components/chat/chat_header.tsx

```diff
@@ -31,19 +31,21 @@ const minWidthClassName = css`
 export function ChatHeader({
   title,
   loading,
-  knowledgeBase,
+  disabled,
   connectors,
+  knowledgeBase,
   onSaveTitle,
 }: {
   title: string;
   loading: boolean;
-  knowledgeBase: UseKnowledgeBaseResult;
+  disabled: boolean;
   connectors: UseGenAIConnectorsResult;
+  knowledgeBase: UseKnowledgeBaseResult;
   onSaveTitle?: (title: string) => void;
 }) {
   const hasTitle = !!title;
 
-  const displayedTitle = title || EMPTY_CONVERSATION_TITLE;
+  const displayedTitle = disabled ? UPGRADE_LICENSE_TITLE : title || EMPTY_CONVERSATION_TITLE;
```

Comment:
can we change disabled to `licenseInvalid` if we use it for changing the title? or we can use `disabled` and set the title as a prop